.vitepress/theme/components/Helpers/CardLink.vue

```diff
@@ -102,8 +102,7 @@ export default {
     min-width: 100px;
     height: 64px;
     &:not(.has-bg)::before {
-      -webkit-mask-size: 32px;
-              mask-size: 32px;
+      mask-size: 32px;
```

Comment:
Removing the `-webkit-mask` prefixes because:

- the unprefixed version finally shipped in Chrome 120 (current is 123);
- also we have some autoprefixing going on that will add those prefixes at build time anyway (unless we go out of our way to add a `.browserslistrc` to narrow down our target browsers to Chrome 120+, Safari 15.4+ and Firefox 53+).